impl/src/metadata.rs

```diff
@@ -491,6 +485,26 @@ impl Default for RazeMetadataFetcher {
   }
 }
 
+/// Locates a lockfile for the associated crate. A `Cargo.raze.lock` file in the
+/// [RazeSettings::workspace_path](crate::settings::RazeSettings::workspace_path)
+/// direcotry will take precidence over a standard `Cargo.lock` file.
+pub fn find_lockfile(cargo_workspace_root: &Path, raze_output_dir: &Path) -> Option<PathBuf> {
```

Comment:
I feel like this should be in the util package, not the metadata package, because it's not directly related to loading metadata.

Reply:
Updated